OpenSim/Simulation/SimbodyEngine/Test/testJoints.cpp

```diff
@@ -2226,6 +2231,42 @@ void testAutomaticJointReversal()
     ASSERT_EQUAL(accErr, 0.0, sqrt(integ_accuracy));
 }
 
+void testUserJointReversal()
+{
+    using namespace OpenSim;
+
+    cout << "\n==========================================================="
+         << "\n Test joint reversal set by user in old model file"
+         << "\n==========================================================="
+         << endl;
+
+    // Open model.
+    auto model = Model("double_pendulum_testReverse.osim");
+    //model.finalizeFromProperties();
+    model.initSystem();
```

Comment:
> If you need to resolve and access connection information you need to call `finalizeConnections()`, which `initSystem()` does as well.

:scream_cat: Sorry, my revised mental model was "load a model and then call `finalizeFromProperties()` and you're safe". Yes, I probably just needed `finalizeConnections()` (checking now…).